### ISSUE TITLE
Accessibility improvements on Select component

### ISSUE THREAD

**Issue by muratcorlu** [Trendyol/baklava#288]
- [ ] Select component should use related `aria-` attributes
- [ ] Select input should be usable with the keyboard. We need to follow a11y standards for the navigation approach.

**Comment by AykutSarac** [comment]
Maybe we can add "validation as form element" here?

**Comment by muratcorlu** [comment]
@AykutSarac I thought requirements for using select in a form can be covered in a separate issue as in #287 Does it also make sense to you?
